src/main/java/com/example/demo/hibernateConfig/HibernateConfiguration.java

```diff
@@ -49,10 +49,12 @@ public DataSource getDataSource() {
 	}
 	
 	
+	
 	private Properties hibernateProperties() {
 		Properties properties = new Properties();
 		properties.put("hibernate.dialect", dialect);
-		properties.put("hibernate.hbm2dll.auto", "update");
+		properties.put("hibernate.dll-auto", "update");
+		properties.put("generate-ddl", "true");
```

Comment:
W zasadzie tak też się da, ale bym na to nie wpadła.

Można, a nawet powinno się skorzystać z zakomentowanego pliku application properties i dodac tam 
spring.jpa.hibernate.ddl-auto=update
spring.datasource.initialization-mode=never

Reply:
Aplikacja korzysta z application properties. Nie usunąłem klasy konfiguracyjnej, ale ma Configuration wzięty wiec nie ma wpływu na aplikację.